integration_test/support/migration.exs

```diff
@@ -80,6 +81,10 @@ defmodule Ecto.Integration.Migration do
       timestamps
     end
 
+    alter table(:users) do
```

Comment:
We don't necessarily need to do separate calls but we do need to generate a special SQL for postgres. So we need to traverse the alter table instructions, finding all rename commands, and then group all queries by using ";" as separator. If that doesn't work, we can try BEGIN END too:

```sql
BEGIN
  ALTER TABLE foo rename 1;
  ALTER TABLE foo rename 2;
  ALTER TABLE foo others
END;
```

Hopefully one of those ideas work. :)